package.json

```diff
@@ -5,12 +5,14 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "peerDependencies": {
-    "@sinclair/typebox": "^0.25.9",
     "fastify": "^4.0.0"
   },
+  "dependencies": {
+    "@sinclair/typebox": "^0.26.4"
```

Comment:
why this dependency is moved to from `peerDependencies` to `dependencies`?

Reply:
@rvitaliy See rationales on issue https://github.com/fastify/fastify-type-provider-typebox/issues/53

Reply:
ok, i see the issue discussion, personally disagree with this approach.
a dependency should never exposed from a wrapper library.
`@sinclair/typebox` is "unstable" since they don't tag a stable version (>=1.0.0) yet so they minor can mutate public interface, but hotfix must don't change them. so the solution is to allow only versions that we know are compatible with this package.

using `Tilde` instead of `Caret Ranges` https://github.com/npm/node-semver#tilde-ranges-123-12-1 to indicate what version of `@sinclair/typebox` this library support.
in this way if `@sinclair/typebox` change public interfaces during upgrade from `0.25.*` to `0.26.*` this library trigger an alert when final user install an incompatible version and the library have more control about the supported version.

Reply:
sorry i read the documentation again, the `Caret Ranges` under the stable version works like `Tilde` so you already have the control about supported version
![Screenshot 2023-03-29 at 10 18 31](https://user-images.githubusercontent.com/1823711/228471507-8de49720-b6e5-4e23-969f-e1ca95112e0d.png)